Code/Framework/AzToolsFramework/Tests/ArchiveTests.cpp

```diff
@@ -132,76 +152,138 @@ namespace UnitTest
         };
 
 #if AZ_TRAIT_DISABLE_FAILED_ARCHIVE_TESTS
-        TEST_F(ArchiveTest, DISABLED_CreateArchiveBlocking_FilesAtThreeDepths_ArchiveCreated)
+        TEST_F(ArchiveComponentTest, DISABLED_CreateArchive_FilesAtThreeDepths_ArchiveCreated)
```

Comment:
I bet all of the test in these functions no longer need to be disabled anymore.
Since we are not reliant on the Archive applications on the host platform, the test should pass for all the platforms.

Can you remove `#if AZ_TRAIT_DISABLE_FAILED_ARCHIVE_TESTS`  checks and run the test on Jenkins?

Reply:
Will check on the next AR run.

Reply:
Followup:  The tests failed on Linux, so we will have to keep them disabled and spend some time getting them to pass.